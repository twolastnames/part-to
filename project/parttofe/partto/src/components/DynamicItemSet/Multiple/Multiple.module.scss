.multiple {
  position: relative;
  height: 100%;
  width: 100%;
  flex: 1;
  flex-direction: row;
}

.switchButton {
  position: absolute;
  left: -42px;
}
