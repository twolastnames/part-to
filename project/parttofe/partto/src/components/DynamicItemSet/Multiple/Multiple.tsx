import React from "react";

import classes from "./Multiple.module.scss";
import { Item } from "../DynamicItemSet";

export interface MultipleProps {
  items: Array<Item>;
}

export function Multiple({ items }: MultipleProps) {
  return (
    <div className={classes.multiple} data-testid="Multiple">
      WIP
    </div>
  );
}
