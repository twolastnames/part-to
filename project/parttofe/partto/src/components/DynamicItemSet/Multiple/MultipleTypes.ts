import { ContextDescription } from "../../../providers/DynamicItemSetPair";
import { Item } from "../DynamicItemSetTypes";

export interface MultipleProps {
  items: Array<Item>;
  context: ContextDescription;
  pausedByDefault?: boolean;
}
