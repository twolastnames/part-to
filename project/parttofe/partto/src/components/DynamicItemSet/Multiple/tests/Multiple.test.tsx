import React from "react";
import { expect, test } from "@jest/globals";
import { render, screen } from "@testing-library/react";
import { Multiple } from "../Multiple";

test("snapshot", () => {
  render(
    <Multiple
      items={[
        {
          listView: <div>first list view</div>,
          detailView: <div>first detail view</div>,
          itemOperations: [],
        },
        {
          listView: <div>second list view</div>,
          detailView: <div>second detail view</div>,
          itemOperations: [],
        },
      ]}
    />,
  );

  const component = screen.getByTestId("Multiple");
  expect(component).toMatchSnapshot();
});
