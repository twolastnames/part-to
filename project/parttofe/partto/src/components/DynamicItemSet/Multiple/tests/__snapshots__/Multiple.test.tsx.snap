// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  class="multiple"
  data-testid="Multiple"
>
  WIP
</div>
`;
